### ISSUE TITLE
thoughts about ontology connections?

### ISSUE THREAD

**Issue by vjcitn** [bhklab/PharmacoDB#10]
I have been looking through the capacity of Cell Line Ontology and ChEBI to 
give formal definitions of tokens in the cell_line and compounds components.
I think it can be done, although of course there will be gaps.  Are you looking into
this?  I am about to push github.com/vjcitn/RxGeno that demonstrates some of
the issues.

**Comment by vjcitn** [comment]
I see that you are using cellosaurus as the base ontology for cell lines.  I use
the API to extract data, and only the internal IDs seem to be returned.  Can one
retrieve the cellosaurus tags too?

**Comment by p-smirnov** [comment]
@vjcitn I don't believe we have this data exposed but we definitely can implement an endpoint to recover it from PharamcoDB. 

We have tried to work with the CLO several years ago (!~2013) and had mixed results. However the CLO seems to have been under active development since then and I think its definitely worthwhile to try to map our database to the formal ontologies.

**Comment by p-smirnov** [comment]
@assefamaru Can you comment on the retrieving cellosaurus through the API

**Comment by assefamaru** [comment]
@vjcitn @p-smirnov PharmacoDB API does not fully parse/expose cellosaurus data, but an API for cellosaurus has already been implemented in a separate package: https://github.com/assefamaru/cellosaurus-api. Data is extracted from the structured text files provided by Cellosaurus: https://github.com/calipho-sib/cellosaurus/blob/master/cellosaurus.txt.
